### DISCUSSION TITLE
No docs on self-hosting?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [MaterializeInc/materialize#16017] [Q&A] [answered]
Curious why there are no docs on self-hosting Materialize, specifically for clusters. Is it not intended to be self-hosted?

**Comment by bobbyiliev** [comment]
Hi there @danthegoodman1,

The new Materialize platform is currently only delivered as a cloud service, you create a Materialize account and create resources, and Materialize manages the resources, similar in structure to Snowflake or Redshift. As of the time being, this new cloud-native architecture is not feasible to deliver as a self-managed or on-prem service. You can sign up for early access here:

> https://materialize.com/register/

You can still install the old Materialize binary locally, you can find the instructions on how to do that here:

> https://materialize.com/docs/lts

Hope that this helps and let me know if you have any questions.

**Reply by danthegoodman1** [reply]
gotcha, thanks for clarifying, did not realize it was cloud only now!

**Reply by lialzm** [reply]
Will the Materialize binaries be no longer updated?

**Reply by bobbyiliev** [reply]
Hey @lialzm, currently the main focus and all development time are on the new cloud-native version of Materialize. Which is still developed in public, it’s still the same BSL, but it’s only delivered as a cloud service.

If you’re interested in trying it out, we’re currently working through the early access list ([register here](https://materialize.com/register/)). We set everyone up with plenty of free trial credits to learn the new system before deciding if it’s something they want to build on.

You’re free to continue to use the binary, but we have to focus our efforts on the current version.

**Reply by morsapaes** [reply, marked as answer]
We recently announced self-managed Materialize (see [this blogpost](https://materialize.com/blog/self-managed/)). This new deployment option is packaged as a containerized app that can be deployed in K8s, for example using [this Helm chart](https://github.com/MaterializeInc/materialize/pull/30491). Although licensing still applies, we're in Early Access right now and open to chat about your expectations. Above all: we'd love some early feedback on the deployment experience!

**Comment by pykenny** [comment]
What if I want to build everything from source on my machine, given that "*it's free to use on single node*" from licensing? LTS is 0.26 and I'm totally fine with officials not providing binaries (for pushing users to use cloud hosted solution) since then, but source code is currently on 0.43 as the latest version...

**Reply by bobbyiliev** [reply]
In case you may have a need for a local build of Materialize to use in CI/CD, automated testing, etc. for these purposes (and for our own developers) we publish a development image of the current version of Materialize that bundles the distributed services into a single Docker Image. Note that the Docker image is meant for testing and development purposes only, it has completely different performance characteristics and does not scale.

**Comment by morsapaes** [comment]
@danthegoodman1 @pykenny: we recently announced self-managed Materialize (see [this blogpost](https://materialize.com/blog/self-managed/)). This new deployment option is packaged as a containerized app that can be deployed in K8s, for example using [this Helm chart](https://github.com/MaterializeInc/materialize/pull/30491). Although licensing still applies, we're in Early Access right now and open to chat about your expectations. Above all: we'd love some early feedback on the deployment experience!

**Comment by kleenkanteen** [comment]
The docs are here btw: https://materialize.com/docs/self-managed/v25.1/installation/

It wasn't clear to me how to find it either, but it turns out this dropdown on the top left of the docs page gives you a self-managed option: 
<img width="524" alt="image" src="https://github.com/user-attachments/assets/83ffff76-3eab-4fa1-a188-d27c23e41255" />

You can also ask kapa.ai (topmost button in my pic), and it will redirect you to the self-managed page.
